tests/plugins/mocha.test.support/hooks/hook.js

```diff
@@ -32,24 +33,66 @@ exports.init = (logger, config, cli) => {
 
 	cli.on('build.post.compile', async (builder, done) => {
 		if (builder.platformName === 'android') {
-			await wakeDevices(logger, builder).catch(e => logger.warn(`could not wake ${builder.deviceId}: ${e}`));
+			const emulatorManager = new EmulatorManager(config);
+
+			await wakeDevices(logger, builder, emulatorManager)
+				.catch(e => logger.warn(`Could not wake ${builder.deviceId}: ${e}`));
+			await enableAutoRotation(logger, builder, emulatorManager)
+				.catch(e => logger.warn(`Could not enable auto-rotation ${builder.deviceId}: ${e}`));
 		}
 		done();
 	});
 };
 
-async function adb(argumentArray) {
-	return new Promise(resolve => appc.subprocess.run(ADB_PATH, argumentArray, { shell: false, windowsHide: true }, (error, stdout, _stderr) => {
-		resolve(stdout);
-	}));
+async function adb(args) {
```

Comment:
I don't which exact change broke the build, but the CI machine is waiting after spitting out that it's going to wake the emulator and then not proceeding and the build eventually times out. I suspect perhaps the changes to running `adb` might be involved?